spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configuration-watcher/src/test/java/org/springframework/cloud/kubernetes/configuration/watcher/HttpBasedConfigMapWatchChangeDetectorTests.java

```diff
@@ -21,20 +21,28 @@
 import java.util.List;
 import java.util.Map;
 
+import com.github.tomakehurst.wiremock.WireMockServer;
```

Comment:
from here and down two things happen:

- move to junit5
- enable tests back that were commented